deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/childrenChartInstaller/installJob.yaml

```diff
@@ -24,8 +24,8 @@ spec:
       restartPolicy: Never
       containers:
         - name: helm-installer
-          image: {{ .Values.installer.image.repository }}:{{ .Values.installer.image.tag }}
-          imagePullPolicy: {{ .Values.installer.image.pullPolicy }}
+          image: {{ .Values.images.migrationConsole.repository}}:{{ .Values.images.migrationConsole.tag }}
+          imagePullPolicy: {{ .Values.images.migrationConsole.pullPolicy }}
```

Comment:
I'm on the fence about this one.  The values could define the installer image as the migration console.  We don't need the migration console, just a lightweight POSIX environment (IIRC, we don't need java or even python).  This also puts a dependency on the migration console image that might not need to be there.

It might not be a concern for us, but it could be a concern for external parties that would want to use their own image here (like alpine).